recipes/pycraf-gui/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "pycraf-gui" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pycraf-gui-{{ version }}.tar.gz
+  sha256: 1403ac72528b2f4600e89d4a1746888162cadc734490d819f2c08c6a4d1a4910
+
+build:
+  entry_points:
+    - pycraf-gui = pycraf_gui.__main__:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+    - setuptools
+    - setuptools_scm
+    - astropy
+
+  run:
+    - python >=3.8
+    - astropy
+    - numpy
+    - matplotlib >=3.3
+    - pycraf
+    - pyqt
+    - pytest
+    - pytest-astropy
+    - pytest-qt
```

Comment:
pycraf and pycraf-gui make use of astropy's [TestRunner](https://docs.astropy.org/en/latest/testhelpers.html#astropy-test-runner), which allows to run the test via a function defined within the module. It may seem a bit overkill, but most of our users don't know much about programming etc., and will never run pytest standalone.

Reply:
Ahh ok makes sense then.